packages/driver/src/cy/commands/navigation.coffee

```diff
@@ -304,15 +304,14 @@ module.exports = (Commands, Cypress, cy, state, config) ->
         when not resp.isOkStatusCode
           err = new Error
           err.gotResponse = true
-          _.extend(err, resp)
-
+          _.extend(err, resp, { originalUrl: url});
```

Comment:
@jennifer-shehane When do we expect the error message to be `/foo.html` vs `http://localhost:3500/whatever`?
I changed this code because with the query param, instead of the error throwing `404 from http://localhost:3500?q=foo`
it would throw `404 from http://localhost:3500/?q=foo` even though the request was sent without the slash. Error message, I'm referring to is the "visit.loading_type_failed" message